src/styles/blocs/product_card.css

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+  margin-bottom: 40px;
+
+  display: block;
+}
+
+.card__title {
+  display: block;
+
+  margin-bottom: 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  line-height: 14px;
+
+  margin-bottom: 16px;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+  line-height: 18px;
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.card__buy-button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 14px;
+  text-transform: uppercase;
+  color: #fff;
+
+  margin: 0 auto;
+  padding: 10px 68px 11px;
```

Comment:
Move it to block-properties (before fonts)

Reply:
didn't quite understand where to move it to